com.stansassets.scene-management/BuildConfigurator/Runtime/BuildConfiguration.cs

```diff
@@ -125,6 +132,14 @@ internal bool IsSceneAddressable(string sceneName)
             return false;
         }
 
+        internal void RemoveAddressablesPreferences()
```

Comment:
```suggestion
        internal void CleanEditorPrefsData()
```
Please, consider renaming this method.